book/website/communication/citable/citable-cff.md

```diff
@@ -0,0 +1,153 @@
+repository-code: "https://github.com/citation-file-format/my-research-software"
+title: "My Research Software"
+version: 0.11.2
+```
+
+The complete list of fields is described in the [CFF schema guide](https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md). In the next section, you can find out which tools can help you create and use the `CITATION.cff` file.
+
+### Steps to Make Your Software Citable
+
+To make your software citable, you need to follow the three steps below.
+
+#### Step 1. Create a `CITATION.cff` File
+
+There are two ways of creating a `CITATION.cff` file.
+
+1. Edit the file manually in your favorite code editor. The disadvantages of this method are installing the required tools on your system and doing the validation yourself. The error messages of the validation can be relatively long and difficult to understand.
+
+2. Use [cffinit](https://citation-file-format.github.io/cff-initializer-javascript/) which guides you through the process of creating your citation file. The `cffinit` has a few advantages compared to manual editing such as
+
+    - no need for installing extra tools;
+    - no need for automatic validation;
+    - guidance for each fields;
+    - visual feedback to indicate issues.
+
+We suggest using `cffinit` as it simplifies the creation and validation. For more details on using `cffinit` see {ref}`cm-citable-cffinit`.
+
+#### Step 2. Validate Your `CITATION.cff` File
+
+```{note}
+If you used `cffinit` to create your `CITATION.cff`, then you can skip to [step 3](citable-cff.html#add-your-citation-cff-to-a-public-code-repository).
+```
+
+Once you have a `CITATION.cff` file, it needs to be validated to make sure there are no issues. You can validate your `CITATION.cff` file on the command line with the [`cffconvert` Python package](https://pypi.org/project/cffconvert/).
+
+```shell
+cd <directory-containing-your-CITATION.cff>
+cffconvert --validate
+```
+
+If you prefer to use Docker, you can use the [`cffconvert` Docker image](https://hub.docker.com/r/citationcff/cffconvert):
+
+```bash
+cd <directory-containing-your-CITATION.cff>
+docker run --rm -v ${PWD}:/app citationcff/cffconvert --validate
+```
+
+If you get error messages, look for the relevant validation error and fix it.
+
+To make sure your GitHub repository always has a valid `CITATION.cff` file, you can use the GitHub Action [cff-validator](https://github.com/marketplace/actions/cff-validator).
+
+#### Step 3. Add Your `CITATION.cff` to a Public Code Repository
+
+After creating a `CITATION.cff` file, you will need to add it to your GitHub repository so that GitHub can recognize it. GitHub already has instructions to show how to do this.For instructions please see [GitHub's guide on software citation.](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files)
```

Comment:
Split into three lines. And move the full stop from inside the square brackets to outside the round brackets.

Reply:
done